Source/Blazorise/Base/BaseInputComponent.razor.cs

```diff
@@ -388,7 +388,7 @@ protected string CurrentValueAsString
     /// <summary>
     /// Gets the size based on the theme settings.
     /// </summary>
-    protected Size ThemeSize => Size.GetValueOrDefault( Theme?.InputOptions?.Size ?? Blazorise.Size.Default );
+    protected Size ThemeSize => ( Size ?? ParentAddons?.Size ).GetValueOrDefault( Theme?.InputOptions?.Size ?? Blazorise.Size.Default );
```

Comment:
Following your previous implementation wouldn't this be:
`Size.GetValueOrDefault( ParentAddons?.Size ?? Theme?.InputOptions?.Size ?? Blazorise.Size.Default );`
Which would be a bit more readable?